src/github.com/getlantern/enproxy/conn_impl_reads.go

```diff
@@ -4,101 +4,86 @@ import (
 	"fmt"
 	"io"
 	"net/http"
-	"time"
 )
 
 // processReads processes read requests by polling the proxy with GET requests
 // and reading the data from the resulting response body
-func (c *Conn) processReads() {
-	// Wait for connection and response from first write request so that we know
-	// where to send read requests.
-	initialResponse := <-c.initialResponseCh
-	err := initialResponse.err
-	if err != nil {
-		return
-	}
-
-	proxyHost := initialResponse.proxyHost
-	proxyConn := initialResponse.proxyConn
-	resp := initialResponse.resp
+func (c *conn) processReads() {
+	increment(&reading)
```

Comment:
You'll see a lot of these calls to increment and decrement.  This is used to generate the periodic metrics printout for debugging connections.